module-cellular/at/cmd/src/CFUN.cpp

```diff
@@ -13,7 +13,7 @@ namespace at
     namespace cmd
     {
         using namespace std::chrono_literals;
-        CFUN::CFUN(at::cmd::Modifier mod) noexcept : Cmd("AT+CFUN", mod, 150s)
+        CFUN::CFUN(at::cmd::Modifier mod) noexcept : Cmd("AT+CFUN", mod, 17s)
```

Comment:
Maybe it would be nice to make a variable from this 17s....?

Reply:
is this value confirmed with quectel?

Reply:
Yes, it is